src/main/java/org/olf/folio/order/OrderImport.java

```diff
@@ -245,54 +248,128 @@ public  JSONArray  upload(String fileName) throws IOException, InterruptedExcept
 				fundDist.put("value", 100);
 				fundDist.put("fundId", fundId);
 				funds.put(fundDist);
 				orderLine.put("fundDistribution", funds);
+				String numRecString = new String();
+				if (numRec < 10 ) {
+					numRecString = "0"+ String.valueOf(numRec);
+				} else {
+					numRecString = String.valueOf(numRec);
+				} 
+					
+				String poLineNumber = "a" + poNumberObj.get("poNumber") + "-"+ numRecString;
+				orderLine.put("poLineNumber", poLineNumber);
 				orderLine.put("purchaseOrderId", orderUUID.toString());
 				poLines.put(orderLine);
-				order.put("compositePoLines", poLines);
+				order.put("compositePoLines", poLines);				
+			
+			} catch(Exception e) {
+				logger.error(e.toString());
+				JSONObject responseMessage = new JSONObject();
+				responseMessage.put("error",e.toString());
+				responseMessage.put("PONumber", poNumber);
+				responseMessages.put(responseMessage);
+				return responseMessages;
+			}
+			numRec++;
+		} 
+		
+		logger.debug("Here is the PO order number: "+ poNumberObj.get("poNumber"));
+		logger.debug(order.toString(3));
+		
+		//POST THE ORDER AND LINE:
+		String orderResponse = apiService.callApiPostWithUtf8(baseOkapEndpoint + "orders/composite-orders", order, token); 
+		JSONObject approvedOrder = new JSONObject(orderResponse);
+		 
+		// get approved order
+		
+		//GET THE UPDATED PURCHASE ORDER FROM THE API AND PULL OUT THE ID FOR THE INSTANCE FOLIO CREATED:
+		logger.debug("getUpdatedPurchaseOrder");
+		String updatedPurchaseOrder = apiService.callApiGet(baseOkapEndpoint + "orders/composite-orders/" +orderUUID.toString() ,token); 
+		JSONObject updatedPurchaseOrderJson = new JSONObject(updatedPurchaseOrder);
+		logger.debug("updated purchase order...");
+		logger.debug(updatedPurchaseOrderJson.toString(3));
+		
+		//JSONObject poMessage = new JSONObject();
+		//poMessage.put("poNum", poNumberObj.get("poNumber"));
+		//responseMessages.put(poMessage);
+		
+		// read through again.
+		FileInputStream in2 = new FileInputStream(filePath + fileName);
+		MarcReader reader2 = new MarcStreamReader(in2);
+		numRec = 0; 
+		
+		while (reader2.hasNext()) {
+			try {
+				 
+				record = reader2.next();				
+				JSONObject responseMessage = new JSONObject();
+				responseMessage.put("poNumber", poNumberObj.get("poNumber"));				 
+				
+				DataField twoFourFive = (DataField) record.getVariableField("245");
+				DataField nineEighty = (DataField) record.getVariableField("980");
+			    DataField nineFiveTwo = (DataField) record.getVariableField("952"); 
+			    
+				String title = marcUtils.getTitle(twoFourFive);
+				responseMessage.put("title", title);
 				
-				//POST THE ORDER AND LINE:
-				String orderResponse = apiService.callApiPostWithUtf8(baseOkapEndpoint + "orders/composite-orders",order,token); 
-				JSONObject approvedOrder = new JSONObject(orderResponse);
-				logger.info(orderResponse);
+				String notes =  marcUtils.getInternalNotes(nineEighty);
```

Comment:
There was previously an issue about adding the content of Notes but the issue was redacted.  I saw no reason to remove the method, even if unused, in case that issues is re-created.